### ISSUE TITLE
Update runtime pack package names

### ISSUE THREAD

**Issue by dsplaisted** [dotnet/core-setup#6508]
We should switch to the following package IDs for the runtime packs:

- `Microsoft.NETCore.App.Runtime.<RID>`
- `Microsoft.WindowsDesktop.App.Runtime.<RID>`
- `Microsoft.NETCore.DotNetHostResolver.<RID>`
- `Microsoft.NETCore.DotNetHostPolicy.<RID>`

I don't think we have previously discussed the ID format for the host resolve and host policy apps, so we could do something like dropping the `DotNet` and/or adding `.App` to them for more consistency with the other packages.

@dagood @ericstj @nguerrera

**Comment by nguerrera** [comment]
I thought the plan was to flatten the runtime pack, such that the last two are subsumed by the first one.

**Comment by nguerrera** [comment]
I think this amounts to a dupe for creating the real runtime packs, which still isn't in for base framework and windows desktop.

**Comment by dsplaisted** [comment]
> I thought the plan was to flatten the runtime pack, such that the last two are subsumed by the first one.

Good point.  I think we discussed it and there may have been a reason not to merge them, but maybe I'm confusing with the AppHost pack which we want to keep separate since it's used separately from the runtime pack in some cases.

**Comment by dagood** [comment]
Yeah, runtime pack names are covered by https://github.com/dotnet/core-setup/issues/4775 (netcoreapp) and https://github.com/dotnet/core-setup/issues/5073 (windowsdesktop).

I actually had the HostPolicy merge it in the checklist at https://github.com/dotnet/core-setup/issues/4775, so I added a line for the HostResolver to make sure that gets in too. It came up on 1/18, mail thread "Next steps for .NET Core 3 targeting packs", pointing out that [GenerateBundledVersions.targets#L206](https://github.com/dotnet/core-sdk/blob/27a5cc58b3542c6d70a040ac28689559a612bb43/src/redist/targets/GenerateBundledVersions.targets#L206) should collapse to just the runtime pack. You both agreed on it, nobody objected, and I haven't seen anything come up to suggest it shouldn't happen, so it sounds good to me.

Closing: accounted for by other issues.
